src/sonic-yang-models/yang-templates/sonic-acl.yang.j2

```diff
@@ -202,13 +202,13 @@ module sonic-acl {
 						when "not(IP_TYPE) or boolean(IP_TYPE[.='ANY' or .='IP' or .='IPV4' or .='IPv4ANY' or .='ARP'])";
 						leaf ICMP_TYPE {
 							type uint8 {
-								range 1..44;
+								range 0..44;
```

Comment:
Maximum value is 44, why do we have this limit?

Reply:
Yes, maximun value is 44. We have this limit.

Reply:
https://www.iana.org/assignments/icmp-parameters/icmp-parameters.xhtml
https://www.iana.org/assignments/icmpv6-parameters/icmpv6-parameters.xml
Please check above links, do we need to modify the type range and code range?

Reply:
Updated.